site-local/config

```diff
@@ -0,0 +1,8 @@
+BRAND_NAME=Dasharo
```

Comment:
branding/ and site-local shoudl remain in fork AFAIK.

You should base this port on files from upstream linuxboot/heads, not dasharo/heads fork (although the rest of the files are the same)

Reply:
See https://github.com/Dasharo/heads/pull/13

Reply:
@filipleple cc